### ISSUE TITLE
⚠️⚠️ URGENT - DOKAN SUBSCRIPTION - Stripe Gateway Fee is not registered in the Dokan > Reports

### ISSUE THREAD

**Issue by Sophie-2e** [getdokan/dokan#2187]
### Bug Description

Customer Support Ticket: #40285, asked me to open the issue.

Issue probably related to Stripe Express again:

DOKAN SUBSCRIPTION - Stripe Gateway Fee is not registered in the Dokan > Reports but registered in Stripe : https://share.vidyard.com/watch/mdnK9yPJPDjZgVxzKwpP1e?

In this environment, the admin is paying the stripe processing fee (in Woocommerce > Settings > Payments > Dokan Stripe Express > Take Processing Fees from Sellers > Option not activated).

### Step To Reproduce

https://share.vidyard.com/watch/mdnK9yPJPDjZgVxzKwpP1e?

### Expected Behaviour

https://share.vidyard.com/watch/mdnK9yPJPDjZgVxzKwpP1e?

### Actual Behaviour

https://share.vidyard.com/watch/mdnK9yPJPDjZgVxzKwpP1e?

### Additional Information

https://share.vidyard.com/watch/mdnK9yPJPDjZgVxzKwpP1e?

### Environment (please complete the following information)

```shell
1. Dokan Version: [3.9.10]
2. Dokan Pro Version: [3.9.10]
3. WooCommerce Version: [8.6.1]
4. Browser [all]
5. Current Theme: [all]
```

**Comment by Sophie-2e** [comment]
Hello, still not working after Dokan 3.10 > update

**Comment by Sophie-2e** [comment]
Hey Dokan team, I also confirm that it has been fixed in the last Dokan 3.10.2 update, thank you so much for your incredible work @shohag121 > You rock! Such a troubleshooter 👌🎉
